Sources/NIOSSL/TLSConfiguration.swift

```diff
@@ -276,6 +280,9 @@ public struct TLSConfiguration {
 
     /// The private key associated with the leaf certificate.
     public var privateKey: NIOSSLPrivateKeySource?
+    
+    /// PSK Callback to get the key and identity from either the client or server.
+    public var pskCallback: PSKIdentityCallbackManager? = nil
```

Comment:
We want to let users configure the callbacks, not the manager. In this instance we're happy to let them configure both at once.

Reply:
Resolved in 516a89f.